pixi.toml

```diff
@@ -43,10 +45,13 @@ test-common-wheels-ci = { cmd = "pytest --numprocesses=auto --verbose tests/whee
 test-common-wheels-dev = { cmd = "pytest --numprocesses=auto tests/wheel_tests/", depends-on = [
   "build",
 ] }
-test-integration-ci = "pytest --numprocesses=auto --durations=10 --verbose tests/integration"
-test-integration-dev = { cmd = "pytest --numprocesses=auto --durations=10 tests/integration", depends-on = [
+test-integration-ci = "pytest --runslow --numprocesses=auto --durations=10 --verbose tests/integration"
```

Comment:
I think we can simplify this and just run all the tests.

In case we need to filter them, we can use pytest -m 'not slow'.

In this case we can remove `pytest_configure` and `pytest_collection_modifyitems`

Reply:
Your suggestion worked like a charm 🎉